### ISSUE TITLE
Email notifications are not descriptive enough

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#7341]
### Steps to reproduce
1. share files with people on https://ocis.ocis-traefik.latest.owncloud.works/
2. open https://mail.ocis-traefik.latest.owncloud.works/
3. read notification mails

```
Received: from localhost ([172.18.0.6]) by inbucket
  for <marie@example.org>; Tue, 26 Jul 2022 06:18:37 +0000 (UTC)
Content-Transfer-Encoding: quoted-printable
Date: Tue, 26 Jul 2022 06:18:37 +0000
Mime-Version: 1.0
From: <notifications@ocis.ocis-traefik.latest.owncloud.works>
To: <marie@example.org>
Content-Type: text/plain; charset=UTF-8

You got a share!
```

example:
![Screenshot 000101](https://user-images.githubusercontent.com/26610733/181041275-06984f3c-3bd0-4aee-a471-54d36d264a97.png)

### Expected behaviour
- Incorporate learnings from oc10 mails
- Mails should be enriched by:
  - clear sender name
  - useful subject
  - useful description
  - link in description
  - imprint
  - maybe HTML mail

**Comment by JanAckermann** [comment]
@tbsbdr Afaik this should be handled by the backend not the frontend?

**Comment by tbsbdr** [comment]
sure - this is mainly about wording and content

**Comment by JanAckermann** [comment]
Fair enough, but I don't think this is something we can change in web, those things have to be done in ocis.

**Comment by tbsbdr** [comment]
update: updated expected behaviour

**Comment by kulmann** [comment]
Can we move this issue to the ocis repo? Has nothing to do with web.

Also: the backend knows the language that was chosen by the user (otherwise defaults to english,  browser language is not known in the backend). But easiest start still is just sending english + german in one email like you proposed.

**Comment by tbsbdr** [comment]
okok, I'll move
